src/backend/distributed/test/run_from_same_connection.c

```diff
@@ -200,11 +200,12 @@ GetRemoteProcessId()
 
 	appendStringInfo(queryStringInfo, GET_PROCESS_ID);
 
-	ExecuteOptionalRemoteCommand(singleConnection, queryStringInfo->data, &result);
+	int queryResult = ExecuteOptionalRemoteCommand(singleConnection,
+												   queryStringInfo->data, &result);
 
 	int64 rowCount = PQntuples(result);
 
-	if (rowCount != 1)
+	if (queryResult != RESPONSE_OKAY || rowCount != 1)
```

Comment:
This extra check may seem redundant, however this is better than suppressing messages.

Reply:
if we are adding this should we add this check before we call Pqntuples? 

```c
if(queryResult != RESPONSE_OKAY) {
 PG_RETURN_VOID();
} 
int64 rowCount = PQntuples(result);;
```